0.1.0

wkok opened the pull request:
This PR prepares release 0.1.0 that bumps dependencies required to support

- `web` package v1.1
- Flutter v3.27

Commit:
Bumped devtools_extensions to 0.3.0-dev.0

Commit:
Declared package:web as direct dependency

Commit:
Updated readme

Commit:
Bump version to 0.1.0 and update dependencies to support web v1.1

psengelke commented:
I've tested this build on a project using `Flutter v3.27.2` and `web v1.1.0`. Everything seems to work as expected.

I picked up one small warning (that we can do something about) while running `bb build-copy`:
```shell
Expected to find fonts for (MaterialIcons, packages/cupertino_icons/CupertinoIcons), but found (MaterialIcons). This usually means you are referring to font families in an IconData class but not including them in
the assets section of your pubspec.yaml, are missing the package that would include them, or are missing "uses-material-design: true".
```
This can be resolved by adding `cupertino_icons` to the direct dependencies. These font assets are tree-shaken so they will have negligible impact on the build size.

psengelke committed:
Added `cupertino_icons` as a direct dependency.

https://github.com/htihospitality/re-dash-inspector/pull/1#issuecomment-2591760466

psengelke commented:
RE: https://github.com/htihospitality/re-dash-inspector/pull/1#issuecomment-2591760466

I've added `cupertino_icons` as a direct dependency to resolve the warning.

wkok commented:
Re: https://github.com/htihospitality/re-dash-inspector/pull/1#issuecomment-2591760466

I think this warning is safe to ignore. We're not using MaterialIcons or CupertinoIcons directly as we're making use of the [DevTools UI library](https://pub.dev/documentation/devtools_app_shared/latest/ui/)

I'm a bit reluctant to include dependencies in pubspec.yaml that's not strictly needed, as these might clash with user's dependencies in their apps

psengelke commented:
RE: https://github.com/htihospitality/re-dash-inspector/pull/1#issuecomment-2591838495

Fair enough, I'll revert.

psengelke committed:
Revert "Added `cupertino_icons` as a direct dependency."

This reverts commit 61674cf535c7ecb436a019e33534c51e4ec375c8.

Commit:
Update CONTRIBUTING.md to include instructions for compiling web workers

Commit:
Refactor worker to use web package instead of dart:html - WIP

Commit:
Use js_interop toconvert to JSFunction before passing to worker

Commit:
Refactored from prn-str to pr-str as we do not need the newline

wkok committed:
Merge pull request #2 from htihospitality/worker-web-package

Refactor worker to use web package instead of dart:html

Commit:
Updated changelog
